scripts/clear_chronos_user_id_and_submissions.py

```diff
@@ -0,0 +1,21 @@
+"""
+Clears the `chronos_user_id` field on all OSFUser instances. Needed for switching from one Chronos server to another.
+"""
+import sys
+import logging
+import django
+django.setup()
+
+from osf import models
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
likewise, not using logging, but this would be useful to log items that were deleted.